src/core/parameter/parameter.ts

```diff
@@ -334,26 +340,32 @@ export class Parameter extends Config {
    * @param value number to be converted
    * @returns converted number
    */
-  toFixed(value: number): string {
+  toFixed(value: number, fractionDigits: number = 1): string {
     const valueAsString = value.toString();
-    let fractionDigits = 1;
-    if (valueAsString.includes('.')) {
+    if (valueAsString.includes('.') && fractionDigits > 0) {
       fractionDigits = valueAsString.split('.')[1].length;
     }
     return value.toFixed(fractionDigits);
   }
 
   /**
-   * Write textual code.
+   * Write textual code for Python.
    */
-  toCode(): string {
+  toPythonCode(): string {
     let value: string;
     if (this.isConstant) {
       // Constant value.
-      if (typeof this._value === 'string') {
+      if (this._format === 'integer') {
+        // Integer value
+        value = this.toFixed(this._value as number, 0);
+      } else if (this._format === 'float') {
+        // Float value
+        value = this.toFixed(this._value as number);
+      } else if (typeof this._value === 'string') {
```

Comment:
Can this really be possible? If yes, then we need to update the typing, since it can have only the types
```
(property) Parameter._value: number | boolean | number[]
```
This means, that type would not be correct.

Reply:
(addressed below)